### ISSUE TITLE
Story Link Broken for Other Users in Session

### ISSUE THREAD

**Issue by ElliotSchmelliot** [microsoft/azure-boards-estimate#77]
In an estimate session with multiple stories, clicking the story link at the top seems to always open a link to the first story, rather than the current story. This is only a problem for other users in the session; links seem to work fine for the user that created the session.

For example, when I clicked this link:
![image](https://user-images.githubusercontent.com/2681406/153282229-8ee7cf98-493b-40d3-9694-7b7080257a76.png)

It took me to this URL (first story in the pointing session):
![image](https://user-images.githubusercontent.com/2681406/153282658-e85e0ac9-c788-4fcf-95e3-481d6d11aa4a.png)

**Comment by JohnLudlow** [comment]
I noticed today that if I left-clicked the link, I got the right story. If I did anything else (e.g middle-clicked or right-click and select Open in new tab) I get the first story.

If I hover over  the link, the URL pop-up shown at the bottom of the screen is the wrong story.

It's kind of behaving like a link to the wrong story, but left-clicking fires a different JavaScript event which corrects the URL.

I'm in Chrome 98.0.4758.82
